docs/data/guides/telemetry/telemetry.md

```diff
@@ -6,6 +6,25 @@ packageName: '@mui/x-telemetry'
 
 <p class="description">MUI X Telemetry collects anonymous usage data to help improve the library. This guide walk you through how to opt-in, opt-out, and configure telemetry.</p>
 
+## Why is Telemetry collected?
+
+MUI X is widely used by developers to build highly performant and customizable components. To further elevate the product experience, and design better features, we aim to gain deeper insights from our users.
+
+So far, we've only been relying on manual ways such as surveys and community engagement to gather feedback. Although valuable, these are inherently limited in reach, capturing only a fraction of our user base.
+
+Telemetry enables us to accurately measure usage patterns at scale, enabling data-driven decision-making that helps us improve the product. While we appreciate the insights this information provides, we also know that not everyone wants to send usage data and you can choose to keep the telemetry disabled.
+
+## What is being collected?
+
+We track the following details anonymously:
+
+- Unique session ID: This is generated on each run with UUID.
+- General machine information (operating system, version and whether or not the command was run within CI)
+- Package name and version
+- License info
+- IP
+- Timestamp of the event occurrence.
+
```

Comment:
Might even make sense to make it a callout.
```suggestion

**Privacy Assurance**: We are committed to maintaining your privacy. All data collected is anonymized, ensuring that no personally identifiable information is gathered. We focus solely on understanding usage patterns to better serve our customers and developer community.
```

Reply:
Well, regarding privacy. It seems we can't indeed use IP and say it's anonymous.  

https://github.com/mui/mui-private/issues/791